firefox-ios/firefox-ios-tests/Tests/ClientTests/RemoteSettings/RemoteSettingsFetchConfigTests.swift

```diff
@@ -18,17 +18,17 @@ class RemoteSettingsFetchConfigTests: XCTestCase {
             return
         }
 
-        XCTAssertGreaterThan(config.rules.count, 0, "Expected more than 0 rules in the config")
+        XCTAssertGreaterThan(config.collections.count, 0, "Expected more than 0 collections in the config")
```

Comment:
```suggestion
        XCTAssertGreaterThan(
            config.collections.count,
            0,
            "Expected more than 0 collections in the config"
        )
```
⚠️ nit

Reply:
Hmmm how did that slip but the test passed locally 🤔

Reply:
Aha linting issue haha I thought it was about changing `config.rules.count` 😂